skycoin/skywire
# Fix deadlock in managed tp's `updateStatus`

Did you run `make format && make check`?
Yes
Fixes #692 

 Changes:	
- Fixed deadlock which caused visor going offline in hypervisor UI

How to test this PR:
1. Run visor with VPN server, visor with VPN client and visor with hypervisor on 3 different machines
2. Prepare another machine to access hypervisor UI, access the visors page
3. Start VPN client with killswitch
4. Update visors page, and right after it gets loaded, remove transport from the VPN server side
5. Wait like 20-30 seconds, visor should stay online in the UI
